ls1intum/Artemis
# `Programming exercises`: Reduce response payload size during import

### Checklist
#### General


#### Server
#### Changes affecting Programming Exercises

### Motivation and Context



Importing a programming exercise works, but produced a response payload of > 100MB due to a circular JSON structure, causing issues in the client.

### Description

Removed the "tasks" list from the returned exercise as it's not necessary.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Import a programming exercise into an exam
2. Verify that the response payload from the server has code 200 and is not in the MB-range anymore

### Review Progress





#### Code Review
#### Manual Tests
